spec/support/http_library_adapters.rb

```diff
@@ -227,7 +227,11 @@ def make_http_request(method, url, body = nil, headers = {})
   end
 
   def normalize_request_headers(headers)
-    headers.merge('User-Agent' => [Excon::USER_AGENT])
+    if Excon::VERSION.to_f < 0.63
```

Comment:
```suggestion
    if Excon::VERSION.start_with?("0.63")
```

So two reasons I want this change:

  1. Not all semantic versions can be floats
  2. Since Excon is sadly still unstable (`0.X.Y`) we can't safely say *any* major/minor above `0.63.X` is safe.

Reply:
Could we compare Strings?

```
2.6.3 :001 > '0.6.0' < '0.7.3'
 => true
```

Reply:
I see your point and it seems to work fine.

JFTR: I assume that it is safe for all versions higher then 0.63 because the change will unlikely be reverted (if you read the commit, why they introduced this header). I've tested locally with excon 0.60 and excon 0.72.

Reply:
strictly speaking the correct way of doing this tests is 

```ruby
Gem::Version.new(Excon::VERSION) < Gem::Version.new("0.63")
```

Reply:
I had actually forgotten that Gemspec comes with a version comparison API. I had assumed it didn't.